src/Core/Services/Implementations/RelayPushNotificationService.cs

```diff
@@ -251,4 +252,15 @@ public Task SendPayloadToOrganizationAsync(string orgId, PushType type, object p
     {
         throw new NotImplementedException();
     }
+
+    public async Task PushSyncOrganizationStatusAsync(Organization organization)
+    {
+        var message = new OrganizationStatusPushNotification
+        {
+            OrganizationId = organization.Id,
+            Enabled = organization.Enabled
+        };
+
+        await SendPayloadToOrganizationAsync(organization.Id, PushType.SyncOrganizationStatusChanged, message, true);
```

Comment:
Last argument needs to be false, since we don't want to exclude current device.
```suggestion
        await SendPayloadToOrganizationAsync(organization.Id, PushType.SyncOrganizationStatusChanged, message, false);
```